src/fields/lib/abstract-field.js

```diff
@@ -279,15 +254,8 @@ module.exports = class AbstractField {
    * @returns {Promise.<boolean>}
    */
   canWriteOnCreate(bundle) {
-    if (this._canWriteOnCreate) {
-      return Promise.resolve(this._canWriteOnCreate.call(null, bundle))
-        .then(result => {
-          if (!result) {
-            return Promise.reject(new Restypie.TemplateErrors.FieldNotWritable({
-              key: this.key
-            }));
-          }
-        });
+    if (this._canWriteOnCreateField) {
+      return Promise.resolve(this._canWriteOnCreateField.call(null, bundle));
```

Comment:
Passed in because the context wasn't necessary, but I will add it in case it because useful later